glue/core/data_collection.py

```diff
@@ -126,15 +126,22 @@ def _sync_link_manager(self):
             return
 
         # Avoid circular calls
-        with self._no_sync_link_manager():
+        with self._ignore_link_manager_update():
             self._link_manager.update_externally_derivable_components()
 
     @contextmanager
-    def _no_sync_link_manager(self):
+    def _ignore_link_manager_update(self):
         self._disable_sync_link_manager = True
         yield
         self._disable_sync_link_manager = False
 
+    @contextmanager
+    def delay_link_manager_update(self):
```

Comment:
There is no docstring. Will this show up in API doc?

Reply:
Added